src/Humanizer.Tests/Humanizer.Tests.csproj

```diff
@@ -24,9 +24,10 @@
     </Compile>
   </ItemGroup>
   <ItemGroup>
-    <Content Include="ApiApprover\PublicApiApprovalTest.approve_public_api.approved.txt">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </Content>
+    <ContentWithTargetPath Include="ApiApprover\PublicApiApprovalTest.approve_public_api.approved.txt">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
```

Comment:
This should be PreserveNewest or it breaks incremental builds. Why was this changed?

Reply:
I made this change because the ApiApproved test was failing and I thought that this change would fix it. I was wrong. Reverted in e968e34.